src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -817,12 +824,18 @@ export default class FilingHistoryList extends Vue {
               // use display name for primary document's title
               if (prop === item.name) title = item.displayName
               else title = this.filingTypeToName(prop as FilingTypes, null, true)
+
               const date = this.dateToYyyyMmDd(item.submittedDate)
               const filename = `${this.getIdentifier} ${title} - ${date}.pdf`
               const link = legalFiling[prop] as string
               pushDocument(title, filename, link)
             }
           }
+        } else if (prop === 'uploadedCourtOrder') {
+          const title = this.isRoleStaff ? `${item.displayName} ${item.fileNumber}` : `${item.displayName}`
```

Comment:
Is there a requirement to hide the File Number (court order number) from non-staff users?

Reply:
Yes. UX Pin has client view specified. I will also confirm with Ethan once just to be sure.

![image](https://user-images.githubusercontent.com/116035339/213260871-50af4e0f-8164-4ab5-9d43-a6ec936f7019.png)